_posts/2024-03-29-geospatial-techno.md

```diff
@@ -0,0 +1,89 @@
+---
+author: GeoSpatial Techno
+layout: post
+title: How to style a layer using GeoServer and QGIS
+date: 2024-03-29
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### Using GeoServer and QGIS to style a layer
+In this session, we will explore the "Capabilities of SLD" and "How to create styles" to produce beautiful maps. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/9AUgLFS9CCY)
```

Comment:
Why the quotes here?

Reply:
This is based on the first post, which was written by Mr. Garnett.